### PR TITLE
Fs 2042 prevent commenter scoring

### PR DESCRIPTION
**Pull request by Adam-W1**
[FS-945 - Prevent Commenters from seeing Assessor comments](https://digital.dclg.gov.uk/jira/browse/FS-945)
[FS-1875 - Prevent commenters from seeing submitted scores and justifications](https://digital.dclg.gov.uk/jira/browse/FS-1875)
[FS-2042 - Prevent Commenters from adding Scores or Rationales](https://digital.dclg.gov.uk/jira/browse/FS-2042)

### CODE REVIEW

File: app/assess/routes.py [hunk 1]
```diff
@@ -60,15 +69,26 @@ def display_sub_criteria(
...
 
+    common_template_config = {
+        "role_information": role_information,
+        "current_theme_id": theme_id,
+        "sub_criteria": sub_criteria,
+        "application_id": application_id,
+        "fund": fund,
+        "form": form,
+        "comments": comments,
+    }
```
Comment (tferns) [inline app/assess/routes.py:90]: This is a cool refactor

File: tests/api_data/endpoint_data.json [hunk 2]
```diff
@@ -177,7 +177,7 @@
       },
```
Comment (tferns) [inline tests/api_data/endpoint_data.json:177]: Sorry about this... hope this didn't cause too much trouble fixing.  I'd forgotten to update local data it seems after testing the actual integration

File: tests/test_authorisation.py [hunk 3]
```diff
@@ -194,3 +195,102 @@ def test_authorised_commenter_redirected_to_dashboard(
             in response.data
         )
         assert b'<div class="lead-dashboard-stats">' not in response.data
+
+    @pytest.mark.parametrize(
```
Comment (tferns) [inline tests/test_authorisation.py:199]: Beautiful test coverage

**Review by tferns — APPROVED** [review]
Really great work!

File: tests/api_data/endpoint_data.json (hunk 2 above)
Comment (Adam-W1) [reply tests/api_data/endpoint_data.json:177]: no worries 👍

**Pull request merged**